test/Modules/UngradedModules.jl

```diff
@@ -339,6 +339,27 @@ end
   @test length(FMI) == 4
 end
 
+@testset "Homology of ComplexOfMorphisms{FPModule{FqFieldElem}}" begin
+    F = GF(2)
+    function _rep_mat(n)
+        I = [i for i in 1:n-1 for δ in (0,1)]
+        J = [(i+δ-1) % n + 1 for i in 1:n-1 for δ in (0,1)]
```

Comment:
...I'm afraid you overlooked a `delta` here?...

Otherwise, it seems fine now.